bakerydemo/settings/base.py

```diff
@@ -183,3 +185,12 @@
 
 # Wagtail settings
 WAGTAIL_SITE_NAME = "bakerydemo"
+
+USE_I18N = True
```

Comment:
this is already defined on https://github.com/wagtail/bakerydemo/blob/44cfef774d4680a3b5c5e389da6b3ea81d016640/bakerydemo/settings/base.py#L150